setup.py

```diff
@@ -3,37 +3,41 @@
 
 import os
 
-from setuptools import setup
 
-with open(os.path.join(os.path.dirname(__file__), 'README.rst')) as fp:
+# We may be executed from outside the project dir: `python pex/setup.py ...`, so ensure the
+# `setup.py` dir is on the path and we're working from here so that dist/ and build/ dirs get
+# created here and all files indicated by MANIFEST.in are found.
+__HERE = os.path.realpath(os.path.dirname(__file__))
+os.chdir(__HERE)
+
+
+with open(os.path.join(__HERE, 'README.rst')) as fp:
   LONG_DESCRIPTION = fp.read() + '\n'
 
-with open(os.path.join(os.path.dirname(__file__), 'CHANGES.rst')) as fp:
+
+with open(os.path.join(__HERE, 'CHANGES.rst')) as fp:
   LONG_DESCRIPTION += fp.read()
 
 
-# This seems to be a fairly standard version file pattern.
-#
-# Populates the following variables:
-#   __version__
-#   __setuptools_requirement
-#   __wheel_requirement
-__version__ = ''
-version_py_file = os.path.join(os.path.dirname(__file__), 'pex', 'version.py')
-with open(version_py_file) as version_py:
-  exec(compile(version_py.read(), version_py_file, 'exec'))
+from pex import third_party
+third_party.install(expose=['setuptools', 'wheel'])
```

Comment:
Why must these be exposed here? 

Also, is it possible to get here when `include_wheel=False` so that `wheel` would silently not be exposed?